CHANGELOG.asciidoc

```diff
@@ -20,6 +20,122 @@ https://github.com/elastic/beats/compare/v8.5.2\...v8.5.3[View commits]
 - Fix broken zip URL monitors. NOTE: Zip URL Monitors will be removed in version 8.7 and replaced with project monitors. {pull}33723[33723]
 - Fix handling of long UDP messages in UDP input. {issue}33836[33836] {pull}33837[33837]
 
+==== Known issues
+
+*Heartbeat*
+
+Airgapped users of the beta Elastic Synthetics Integration package and beta private locations in Uptime Monitor Management may observe that `observer.geo.name` is not indexed with their heartbeat documents, resulting in location data missing from the Uptime app.
+
+This issue does not impact clusters with external network access. This issue will be fixed in the next release. 
```

Comment:
The is automatically resolved in 8.6.0 because of two PRs: 1. the PR in the integrations repo that provided the fix and 2. The PR in Kibana that automatically upgrades private location integration policies. Not sure if we want to mention both.